programs/clearing_house/src/state/state.rs

```diff
@@ -27,6 +27,7 @@ pub struct State {
     pub default_market_order_time_in_force: u8,
     pub default_spot_auction_duration: u8,
     pub exchange_status: ExchangeStatus,
+    pub lp_cooldown_time: u64,
```

Comment:
gotta move this up in the struct below the other u64 fields to avoid repr(c) adding unexpected padding